src/modules/app/pickList.js

```diff
@@ -24,7 +24,7 @@ export const isItemAdded = (item, items = []) => {
 
 const addItem = (item, items = []) => {
```

Comment:
This function doesn't just add items now. For instance toggleItem. Rename the existing toggleItem function to something related to that we're marking it.